cloud.py

```diff
@@ -13,20 +13,15 @@ def terms():
 
 @app.route("/sla")
 def sla():
-    return "Service Level Agreement Page <h1>SLA<h1>"
+    return render_template("sla.html")
 
 # @app.route("/faqs")
 # def faqs():
 #     return "Frequently Asked Questions Page <h1>FAQ<h1>"

```

Comment:
Are these to be added in a future PR?

Reply:
Yeh, another page that I am still reviewing to put into the structure